demo/vl-radio-group.html

```diff
@@ -0,0 +1,32 @@
+<html>
+
+<head>
+  <title>vl-radio voorbeeld</title>
+  <script type="module" src="demo.js"></script>
+  <script type="module" src="/src/vl-radio.js"></script>
+  <script type="module" src="/src/vl-radio-group.js"></script>
+
+  <link rel="stylesheet" type="text/css" href="/node_modules/vl-ui-core/dist/style.css" />
+</head>
+
+<body>
+  <h1>vl-ui-radio-group demo</h1>
+  <p>
+    <a href="https://overheid.vlaanderen.be/webuniversum/v3/documentation/forms/vl-ui-radio">
+      Meer voorbeelden en documentatie</a>.
+  </p>
+
+  <div>
+    <h2>Radio</h2>
+    <pre>
+      <code class="line-numbers language-markup">
+      </code>
+    </pre>
+    <div class="demo">
+      <vl-radio-group data-vl-name="group-0">
```

Comment:
Hoeft het dan nog wel dat de gebruiker zelf een naam bepaald? Ik zou radio-group precies gewoon zelf eentje laten bepalen. Aangezien dat in de shadow dom zit zou dat zelfs hard coded kunnen zijn denk ik?

Reply:
Dezelfde opmerking als @Pieter-Beckers dus zal ik inderdaad zo voorzien.

Reply:
Mijn opmerking is eerder: laat de naam volledig weg, zelfs niet te kiezen. Tenslotte: waarom moet een gebruiker dit kiezen als dit enkel interne winkel van de groep is.

Reply:
> Lijkt me inderdaad interessant, misschien zelfs initieel overbodig om een name te kunnen geven?

Deze feedback had ik gegeven op Pieter zijn feedback, dus zal inderdaad zo voorzien worden.